liana/resource/ml/_ml_select_resource.py

```diff
@@ -0,0 +1,72 @@
+from pandas import read_csv
+from numpy import unique
+import pathlib
+from pandas import DataFrame
+
+# TODO merge
+
+def select_ml_resource(met_est_resource_name: str = 'metalinksdb') -> DataFrame:
```

Comment:
ML & MR as abbreviations are very confusing. MR is fine on it's own, but ML should be metabolite_sets or something similar.

Reply:
agreed and changed. how to address the folder naming (sc, sp, mk) so far